modules/server/src/middlewares/xviz-message-to-middleware.js

```diff
@@ -50,45 +50,44 @@ export class XVIZMessageToMiddleware {
 
   callMiddleware(xvizType, msg = {}) {
     this.info(`[> ${xvizType.toUpperCase()}]`);
-
     switch (xvizType) {
       // Connection events
-      case 'connect':
+      case 'CONNECT':
```

Comment:
Should these be constants too?

Reply:
Yes I will do another pass. Anything in a switch case should probably be represented by a constant.

Reply:
Curiously these are not in a spec, but part of the protocol never the less.  Need to make sure these are documented appropriately.